### PR TITLE
Added "Trigger Direct Call" Action Type

### PR DESCRIPTION
**Pull request by janexner**
Add a new Action Type called "Trigger Direct Call", which maps directly to a _satellite.track() call.

#23 

## Motivation and Context

This came up in a discussion in the measure slack about the global variables in the Analytics Extension, and how to apply "global" things in Launch in general.

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All tests pass and I've made any necessary test changes.
- [ ] I have run the extension sandbox successfully.

### CODE REVIEW

File: src/lib/actions/directCall.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * @param {Object} settings Action settings.
+ * @param {string} settings.identifier The identifier of the "Direct Call" Event Type that should
+ * be called.
+ * @param {Object} event The underlying event object that triggered the rule.
+ * @param {Object} event.element The element that the rule was targeting.
+ * @param {Object} event.target The element on which the event occurred.
+ * <code>javascript</code> or <code>html</code>.
+ */
+module.exports = function(settings, event) {
+  if ('undefined' !== typeof settings && settings && 'undefined' !== typeof settings.identifier && settings.identifier) {
```
Comment (dompuiu) [inline src/lib/actions/directCall.js:34]: I would change this to `if (settings && settings.identifier)`.

Comment (janexner) [reply src/lib/actions/directCall.js]: done

**Review by dompuiu — APPROVED** [review]
Could you also update the version in `package.json`?

**Comment by janexner** [comment]
> Could you also update the version in `package.json`?

Also done

**Comment by dompuiu** [comment]
@janexner It seems 5 tests are failing on your forked repo. Can you try to run `npm test` or `npm run test:watch` and see if you can find the problem?

**Comment by janexner** [comment]
> @janexner It seems 5 tests are failing on your forked repo. Can you try to run `npm test` or `npm run test:watch` and see if you can find the problem?

I may be in over my head. There are indeed failures... do you have a pointer to some documentation that helps me to troubleshoot? I'm new to karma and react...

**Comment by dompuiu** [comment]
> > @janexner It seems 5 tests are failing on your forked repo. Can you try to run `npm test` or `npm run test:watch` and see if you can find the problem?
> 
> I may be in over my head. There are indeed failures... do you have a pointer to some documentation that helps me to troubleshoot? I'm new to karma and react...

You can run `npm run test:watch`. Then you find the tests that are failing (there are a ton of warnings, but somewhere in there you should see with red the tests that are failing). Then you go to the test that is failing and change `it` to `fit`. This will make only that test to run over and over. Try to fix it. Once it is fixed, you go to the next one. Let me know if you need more details.

**Pull request merged**